test/types.d.ts

```diff
@@ -31,10 +31,13 @@ type ItFunction<STATE> = ((name: string, inner: (state: STATE) => Promise<void>)
     repeat(n: number): ItFunction<STATE>;
 };
 
-interface WorkerState {
+interface FixtureParameters {
 }
 
-interface FixtureState {
+interface WorkerState extends FixtureParameters {
```

Comment:
Can registerWorkerFixture override fixture parameters? If not, WorkerState shouldn't extend FixtureParameters.

Reply:
It can't, but they are defined on the same level. I'll remove inheritance!